i18n/en-US.yml

```diff
@@ -374,13 +374,13 @@ components:
     NO_TRANSIT_CONNECTION:
       body: >-
         No transit connection was found between your origin and destination on
-        the selected day of service.
-      header: No transit connections
+        the selected day of service, using the vehicle types you selected.
+      header: No transit connections for chosen vehicle types
```

Comment:
Could we keep just "No Transit Connections" for the header? The result here is looking a little wordy & I think the description does a good job of clarifying.
<img width="466" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/115499534/cbfca1bc-cd28-401b-b959-011b0dc84328">

Reply:
Sure!